spec/amp-email-format.md

```diff
@@ -0,0 +1,379 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AMPHTML Email
+
+AMP is a technology commonly known for developing super fast web pages on mobile clients. AMP is in fact a set of HTML tags backed by JavaScript that allow all kinds of functionality with a focus on performance and security.
+
+There are [AMP components](https://www.ampproject.org/docs/reference/components) for everything from carousels, to responsive form elements, to retrieving fresh content from remote endpoints. The AMPHTML Email format provides a subset of AMP components that you can use in email messages. Recipients of AMP emails can view and interact with the AMP components directly in the email message.
+
+## FAQ
+
+* What is the official name of this technology?
+
+    * "AMPHTML Email" is the name of this technology.
+
+* Will this allow people to send JavaScript email?
+
+    * No. Valid AMP documents are not permitted to have their own JavaScript.
+
+* I heard that [AMP is going to support JavaScript](https://github.com/ampproject/amphtml/issues/13471). Will this be incorporated into AMPHTML Email?
+
+    *  No. To maintain users’ expectations of security and privacy, we’ll only allow a conservative subset of AMP functionality.
+
+* Are all AMP components supported?
+
+    * No. AMPHTML Email supports a restricted subset of the AMP components in order to ensure user safety in web-based mail clients.
+
+* Which AMP components will be supported?
+
+    * See the [AMP Components](#amp-components) section in the spec below.
+
+* Will the `amp-ad` component be supported?
+
+    * No.
+
+* Will the `amp-analytics` component by supported?
+
+    * No.
```

Comment:
What about amp-pixel?

Reply:
It's also banned.

Reply:
Then it should be: Will `amp-analytics` and  `amp-pixel` components be supported?

Reply:
Done.